### PR TITLE
Adding instructions to build a Debian Package

### PR DESCRIPTION
**Pull request by stricaud**
It does not build a Debian package that can be pushed to the distribution yet, but it provides
an easy way to have a Debian package for MISP for minimal configuration efforts.

It is installed in /usr/share/misp and there are too many things happening in that directory,
such as logs, instead of being in /var/log/misp/.

However it can be useful to a lot of people, and I will gradually improve it over time.

-- STR

## Generic requirements in order to contribute to MISP:

* One Pull Request per fix/feature/change/...
* Keep the amount of commits per PR as small as possible: if for any reason, you need to fix your commit after the pull request, please squash the changes in one single commit (or tell us why not)
* Always make sure it is mergeable in the default branch (as of today 2016-06-03: branch 2.4)
* Please make sure Travis CI works on this request, or update the test cases if needed
* Any major changes adding a functionality should be disabled by default in the config

#### What does it do?

If it fixes an existing issue, please use github syntax: `#<IssueID>`

#### Questions

- [ ] Does it require a DB change?
- [ ] Are you using it in production?
- [ ] Does it require a change in the API (PyMISP for example)?

#### Release Type:
- [ ] Major
- [ ] Minor
- [X] Patch

### CODE REVIEW

File: app/Model/Attribute.php [hunk 1]
```diff
@@ -660,6 +660,11 @@ private function __alterAttributeCount($event_id, $increment = true)
 
     public function afterSave($created, $options = array())
     {
+       //STR
```
Comment (iglocska) [inline app/Model/Attribute.php]: image

Comment (stricaud) [reply app/Model/Attribute.php]: damn, this was a test I left :( let me remove that

Comment (iglocska) [reply app/Model/Attribute.php]: Thanks :)

**Comment by iglocska** [comment]
Thanks a lot!

**Comment by iglocska** [comment]
Awesome stuff :)

**Pull request merged**